### PR TITLE
Add evaluations for LLAMA2

### PR DESCRIPTION
**Pull request by enigma9981**
This pull request incorporates the metrics from TruthfulQA for LLAMA2, but it excludes the MC1 and MC2 metrics.

### CODE REVIEW

**Comment by enigma9981** [comment]
To manually run it, the repository should contain a .env file in the directory harness/Llama2 structured like this:

```
PROJECT_PATH=/workspace/llm/LLM-TPU
LLAMA2_BMODEL_PATH=${PROJECT_PATH}/bmodels/test2.bmodel
LLAMA2_TOKENIZER_PATH=${PROJECT_PATH}/models/Llama2/support/tokenizer.model
LLAMA2_LIB_PATH=${PROJECT_PATH}/harness/Llama2/build/libllama2.so
LLAMA2_LOGITS_LIB_PATH=${PROJECT_PATH}/harness/Llama2/build/libllama2_logits.so

EVA_QUESTION_PATH=${PROJECT_PATH}/harness/TruthfulQA/TruthfulQA_demo.csv

LLAMA2_BMODEL_LOGITS_PATH=${PROJECT_PATH}/bmodels/test_logits.bmodel

```
The variable ```LLAMA2_BMODEL_PATH``` represents the path to the normal bmodel file, which is identical to the bmodels downloaded from the FTP server, while the variable ```LLAMA2_BMODEL_LOGITS_PATH``` represents the path to the bmodel file that generates logits as the output.
